app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -689,6 +702,17 @@ struct DeviceDelegateOpenXR::State {
           controllersReadyCallback = nullptr;
       }
   }
+
+  void InitPassthrough() {
```

Comment:
Nit: the rest of the file uses `Initialize` so perhaps it's better to make this `InitializePassthrough` as well.

Reply:
Right, will change that.